src/pages/PoolDetails/index.tsx

```diff
@@ -141,26 +140,16 @@ export default function PoolDetailsPage() {
           loading={loading}
         />
         <PoolDetailsStats poolData={poolData} isReversed={isReversed} chainId={chainId} loading={loading} />
-        {(token0 || token1 || loading) &&
-          (loading ? (
-            <LinkColumn data-testid="pdp-links-loading-skeleton">
-              <DetailBubble $height={24} $width={116} />
-              {Array.from({ length: 3 }).map((_, i) => (
-                <Row gap="8px" key={`loading-link-row-${i}`}>
-                  <SmallDetailBubble />
-                  <DetailBubble $width={117} />
-                </Row>
-              ))}
-            </LinkColumn>
-          ) : (
-            <TokenDetailsWrapper>
-              <TokenDetailsHeader>
-                <Trans>Info</Trans>
-              </TokenDetailsHeader>
-              {token0 && <TokenDescription tokenAddress={token0.id} chainId={chainId} />}
-              {token1 && <TokenDescription tokenAddress={token1.id} chainId={chainId} />}
-            </TokenDetailsWrapper>
-          ))}
+        <TokenDetailsWrapper>
+          <TokenDetailsHeader>
+            <Trans>Links</Trans>
+          </TokenDetailsHeader>
```

Comment:
nit: rename to PoolDetailsHeader/Wrapper?

Reply:
This section is specifically referring to the Token Details of the Tokens in the pool not like in TDP.